NodeMCU/HWHandling/HWHandling.ino

```diff
@@ -0,0 +1,74 @@
+#include <ArduinoJson.h>
+#include <WebSocketsClient.h>
+#include <ESP8266WiFi.h>
+
+
+WebSocketsClient NodeWSocket;
+
+#define SERVER "wss://iot-chat-mhashim6.herokuapp.com/"
+#define PORT 80
+#define URL "/"
+
+DynamicJsonDocument doc(200);
+
+void websocketEvent(WStype_t type, uint8_t *data, size_t length) {
+  switch (type) {
+    case (WStype_CONNECTED):
+      Serial.printf("Device is COnnected");
+      NodeWSocket.sendTXT("{\"msgType\":\"auth\",\"userType\":\"HardWare\",\"username\":\"NodeMCU\"}");
+      break;
+
+    case (WStype_TEXT):
+      deserializeJson(doc, data);
+      JsonObject deliveredMessage = doc.as<JsonObject>();
+      //int led = deliveredMessage["led"].as<int>();
+      int led = deliveredMessage[String("led")];
+      int outPutN = led + 4;
+      digitalWrite(outPutN, !digitalRead(outPutN));
+      char *sentMessage;
+      sprintf(sentMessage, "{\"msgType\":\"control\",\"led\":\"%d\"}", outPutN);
```

Comment:
why `outputPin`? it should be `led`. the `frontened` doesn't know about any pins, it only knows that there are 3 leds.

Reply:
Ooops , you are right. it only knows about 0,1, and 2 led numbers , not the pins.